api/src/graphql/users.sdl.js

```diff
@@ -32,4 +32,8 @@ export const schema = gql`
     stripeSubscriptionid: String
     onboarded: Boolean
   }
+
+  type Mutation {
+    updateUser(input: UpdateUserInput!): User
```

Comment:
Can we change this so returning a user is required?

```suggestion
    updateUser(input: UpdateUserInput!): User!
```